charts/osm/templates/cleanup-hook.yaml

```diff
@@ -133,11 +133,11 @@ spec:
         - >
          kubectl delete --ignore-not-found meshconfig -n '{{ include "osm.namespace" . }}' osm-mesh-config;
          kubectl delete crd meshconfigs.config.openservicemesh.io --ignore-not-found;
+         kubectl delete crd multiclusterservices.config.openservicemesh.io --ignore-not-found;
```

Comment:
already present on line 139. Further deleting the CRD will automatically delete the custom resources